include/message_struct.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _MESSAGE_DATA_STATIC_H_
+#define _MESSAGE_DATA_STATIC_H_
+#include "PR/ultratypes.h"
```

Comment:
We prefer to avoid using underscores on header guards for new headers
```suggestion
#ifndef MESSAGE_DATA_STATIC_H
#define MESSAGE_DATA_STATIC_H

#include "PR/ultratypes.h"
```

Reply:
done in acd12b2